site-setup/learn_article.ftl

```diff
@@ -184,181 +167,195 @@ html{box-sizing:border-box;-ms-overflow-style:scrollbar}*,::after,::before{box-s
 	</div>
 </div>
 
-<#attempt>
-	<script>
-		// Table of contents reading indicator
-
-		const headings = document.querySelectorAll('.article-body h2');
+<#noparse>
+	<#attempt>
```

Comment:
Now that you added #noparse we don't need the attempt / recover blocks anymore - can you please remove those?